core/src/test/scala/org/apache/spark/deploy/history/ApplicationCacheSuite.scala

```diff
@@ -128,8 +128,8 @@ class ApplicationCacheSuite extends SparkFunSuite with Logging with MockitoSugar
       completed: Boolean,
       started: Long,
       ended: Long): SparkUI = {
-    val info = new ApplicationInfo(name, name, Some(1), Some(1), Some(1), Some(64),
```

Comment:
It seems that only the parameter pass to the constructor in this test case is not `None`